### ISSUE TITLE
Surveys are in English

### ISSUE THREAD

**Issue by esmeleon** [dib-lab/2018-06-27-SpanishSWC#5]

**Comment by raynamharris** [comment]
You are correct! This is a challenging and complex issue. If we keep it in English, some people won't take it. If we translate it to Spanish, the data won't be included with all the other surveys... I'll assign this to Karen and she can think about how and when to move forward.
